.github/workflows/git-dev.yml

```
@@ -0,0 +1,80 @@
+
+      - name: Generate Coverage Report
+        run: dotnet test # coverage happens by default
+
+      - name: Upload Coverage Report
+        uses: actions/upload-artifact@v4
+        with:
+          name: report
+          path: docfx/coverage/report/
+
+      - name: Pack
+        run: | 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)